dfastmi/gui/dialog_view_model.py

```diff
@@ -367,6 +369,8 @@ def _update_slength(self) -> None:
         except:
             self.slength = "---"
 
+        self.slength_changed.emit(self.slength)
```

Comment:
ain't it better to make slength a property and put the emit in the setter of the property?

Similar as with branch and reach.

Reply:
It cannot be edited in the GUI because it is a calculated value.